### ISSUE TITLE
mkdocs `build` with `--strict` flag complains about `author_profiles` option

### ISSUE THREAD

**Issue by hellt** [squidfunk/mkdocs-material#6408]
### Bug description

```
docker run -v $(pwd):/docs --entrypoint mkdocs ghcr.io/srl-labs/mkdocs-material-insiders:9.4.14-insiders-4.46.0-hellt build --strict
INFO    -  DeprecationWarning: warning_filter doesn't do anything since MkDocs 1.2 and will be removed soon. All messages on the `mkdocs` logger get counted automatically.
  File "/usr/local/lib/python3.11/site-packages/mkdocs_macros/util.py", line 20, in <module>
    from mkdocs.utils import warning_filter
  File "/usr/local/lib/python3.11/site-packages/mkdocs/utils/__init__.py", line 453, in __getattr__
    warnings.warn(

WARNING -  Config value 'plugins': Plugin 'material/blog' option 'author_profiles': Unrecognised configuration name: author_profiles

Aborted with 1 configuration warnings in 'strict' mode!
```

Without `strict` flag build runs

```
❯ docker run -v $(pwd):/docs --entrypoint mkdocs ghcr.io/srl-labs/mkdocs-material-insiders:9.4.14-insiders-4.46.0-hellt build 
INFO    -  DeprecationWarning: warning_filter doesn't do anything since MkDocs 1.2 and will be removed soon. All messages on the `mkdocs` logger get counted automatically.
  File "/usr/local/lib/python3.11/site-packages/mkdocs_macros/util.py", line 20, in <module>
    from mkdocs.utils import warning_filter
  File "/usr/local/lib/python3.11/site-packages/mkdocs/utils/__init__.py", line 453, in __getattr__
    warnings.warn(

WARNING -  Config value 'plugins': Plugin 'material/blog' option 'author_profiles': Unrecognised configuration name: author_profiles
INFO    -  [macros] - Macros arguments: {'module_name': 'main', 'modules': [], 'include_dir': 'macros', 'include_yaml': [], 'j2_block_start_string': '[[[%', 'j2_block_end_string': '%]]]', 'j2_variable_start_string': '[[[', 'j2_variable_end_string': ']]]', 'on_undefined': 'keep', 'on_error_fail': False, 'verbose': False}
INFO    -  [macros] - Extra variables (config file): ['social', 'analytics', 'tags', 'annotate']
INFO    -  [macros] - Extra filters (module): ['pretty']
INFO    -  [macros] - Includes directory: macros
INFO    -  [macros] - Found j2 variable 'block_start_string': '[[[%'
INFO    -  [macros] - Found j2 variable 'block_end_string': '%]]]'
INFO    -  [macros] - Found j2 variable 'variable_start_string': '[[['
INFO    -  [macros] - Found j2 variable 'variable_end_string': ']]]'
INFO    -  git-committers plugin DISABLED
INFO    -  git-committers: found page authors cache file - loading it
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /docs/site
```

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
-

### Reproduction

[9.4.14+insiders.4.46.0-blog-authors-demo.zip](https://github.com/squidfunk/mkdocs-material/files/13498330/9.4.14%2Binsiders.4.46.0-blog-authors-demo.zip)

### Steps to reproduce

Run build with strict and author_profiles option set.

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
THanks for reporting. As mentioned in our documentation, it is called [`authors_profiles`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#adding-author-profiles) (mind the `s`). However I think you might have copied this from here, which was in fact wrong and corrected in d0a536acc.

**Comment by hellt** [comment]
oh my :D haha, ok, that explains it. Thanks
